Dockerfile

```diff
@@ -1,41 +1,84 @@
 # syntax = docker/dockerfile:latest
-FROM python:3.11.5-alpine3.18
-LABEL maintainer='github.com/borgmatic-collective'
-VOLUME /mnt/source
-VOLUME /mnt/borg-repository
+
+FROM python:3.12.2-alpine3.19 as base
+ARG TARGETARCH
+
+LABEL maintainer='modem7'
```

Comment:
Any chance you could change this to `borgmatic-collective` please dude

Reply:
Dun n dun. Also changed the repo in the requirements.txt